modules/network/kademlia/kademlia.js

```diff
@@ -336,12 +346,28 @@ class Kademlia {
                     this.log.error('Failed to join network, will retry in 1 minute');
                     callback(new Error('Failed to join network'));
                 } else {
+                    this.kademliaUtilities.setBootstraps(peers);
+                    this.kademliaUtilities.setRoutingTable(this.node.router);
+
+                    // todo experimental - should be removed
+                    const end = Date.now();
+                    this.log.info(`Network join lasted ${end - start} ms`);
```

Comment:
Should this comment still be here?

Reply:
I think that it's ok to leave this log. But the comments are removed

Reply:
Nice, I meant the comment so great 👍